MarsQA-1/Feature/Languages.cs

```diff
@@ -0,0 +1,160 @@
+using MarsQA_1.Pages;
+using TechTalk.SpecFlow;
+
+namespace MarsQA_1
+{
+    [Binding]
+    public class Languages
+    {
+        [Given(@"All the exsisting records should be cleared")]
+        public void GivenAllTheExsistingRecordsShouldBeCleared()
+        {
```

Comment:
i would also move the steps into a steps folder so that it is easier to find your steps

Reply:
Moved to stepDefinitions folder.